src/Codeception/Module/Symfony.php

```diff
@@ -521,9 +521,15 @@ protected function debugResponse($url)
      */
     private function extractRawRoles(Data $data)
     {
-        $raw = $data->getRawData();
+        // prior to Symfony 3.3
```

Comment:
```
diff --git a/src/Symfony/Component/VarDumper/Cloner/Data.php b/src/Symfony/Component/VarDumper/Cloner/Data.php
index 1c6dd0df86..5a6997cf25 100644
--- a/src/Symfony/Component/VarDumper/Cloner/Data.php
+++ b/src/Symfony/Component/VarDumper/Cloner/Data.php
@@ -16,7 +16,7 @@ use Symfony\Component\VarDumper\Caster\Caster;
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class Data
+class Data implements \ArrayAccess, \Countable, \IteratorAggregate
 {
     private $data;
     private $position = 0;
@@ -34,10 +34,146 @@ class Data
     }
 
     /**
+     * @return string The type of the value.
+     */
+    public function getType()
+    {

```
`getType()` was added with 3.3. On the same release the deprecation warning ws added